Завершаем вёрстку всех компонентов

Alex88GitHub opened the pull request:
---
:mortar_board: [Завершаем вёрстку всех компонентов](https://up.htmlacademy.ru/htmlcss/36/user/2254559/tasks/18)

:boom: https://htmlacademy-htmlcss.github.io/2254559-sedona-36/9/

Alex88GitHub committed:
Декоративные элементы

Исправления №2  по ДЗ

Alex88GitHub committed:
Update styles.css

Правки согласно видео наставника

Alex88GitHub committed:
Формы

Alex88GitHub committed:
Формы

Alex88GitHub committed:
формы

Alex88GitHub committed:
Формы

Alex88GitHub committed:
формы

Alex88GitHub committed:
Update styles.css

Добавляет декоративные эелементы (черточки) в разделы преимуществ.

Alex88GitHub committed:
Update styles.css

Alex88GitHub committed:
формы

Alex88GitHub committed:
Update styles.css

Alex88GitHub committed:
Формы

Alex88GitHub committed:
Формы

Alex88GitHub committed:
Update styles.css

Alex88GitHub committed:
формы

Alex88GitHub committed:
Create triangle-tooltip.svg

Alex88GitHub committed:
Update styles.css

Alex88GitHub committed:
формы

Alex88GitHub committed:
формы

Alex88GitHub committed:
формы

Alex88GitHub committed:
Update styles.css

Alex88GitHub committed:
Правки по пиксель перфект

Alex88GitHub committed:
pixel perfect

4elbik commented on styles/styles.css:
```diff
@@ -615,6 +644,285 @@ main {
+
+.modal-search-button {
+  width: 100%;
+  padding: 18px 256.5px;
```
`.modal-search-button`

1) Нам достаточно указать `padding: 18px 25px;`. Ни к чему нам приплетать сюда польшие отступы + отступы с нецелым числом
2) Со стилями для текста не всё учтено по макету (fz, lh)

Alex88GitHub replied:
Поправил.

4elbik commented on styles/styles.css:
```diff
@@ -615,6 +644,285 @@ main {
+}
+
+.date-title {
+  margin-right: 24px;
```
`.date-title`

Убираем `mr: 24`. Всё)
Остальное кпо сеткам у тебя корректно выставлено. Очень хорошо разбил столбцы!)

Alex88GitHub replied:
Ок, поправил.

4elbik commented on index.html:
```diff
@@ -199,6 +200,72 @@ <h2 class="visually-hidden">Разработчик сайта</h2>
+                      <path d="M3 9v2h14V9H3Z" />
+                    </svg>
+                  </button>
+                  <input class="tourist-number" name="children-number" value="2">
```
`type="number"` здесь и для "Взрослых"

Alex88GitHub replied:
Забыл атрибут значит, поправил.

4elbik replied:
Ещё вкусно было бы докинуть минимальное значение)

Alex88GitHub replied:
Минимум 1 взрослый и 0 детей)

4elbik commented on index.html:
```diff
@@ -199,6 +200,72 @@ <h2 class="visually-hidden">Разработчик сайта</h2>
+                    </svg>
+                  </button>
+                  <input class="tourist-number" name="adult-number" value="2">
+                  <button class="button-tourist-plus">
```
У каждой кнопки в проекте обязательно должен быть прописан `type`. Это не просто так критерий, вот как раз у нас сейчас получается, что кнопки увеличения/уменьшения кол-ва гостей, отправляют форму

Очень, так скажем, наглядный пример, почему необходимо указывать `type`

Alex88GitHub replied:
Понял, тебя. Меня тоже смутило, что кнопки форму отправляют...сейчас понятно стало.

4elbik replied:
Решил только для одной кнопки `type` прописать?)

Alex88GitHub replied:
Видимо уже плохо соображал под вечер и тупанул. Исправил.

4elbik commented on styles/styles.css:
```diff
@@ -684,24 +996,54 @@ main {
+}
+
 .input-price-from,
 .input-price-to  {
```
`.input-price-from, .input-price-to`

Было бы неплохо ещё стилизовать фокусное состояние

Alex88GitHub replied:
сделал outline'ом.

4elbik commented on styles/styles.css:
```diff
@@ -711,9 +1053,61 @@ main {
+}
+
+.toogle-min {
+  top: -10px;
```
Идеальное центрирование асбсолютных элементов:

<img width="1334" alt="image" src="https://user-images.githubusercontent.com/21055475/224528786-b88bcbc6-a350-4a0f-9932-1b1bf88fd3f6.png">

Alex88GitHub replied:
Это да, главное я дальше где-то так и делал, а тут почему-то так не гибко оформил. Поправил.

Alex88GitHub committed:
Корректировки ДЗ №9

Alex88GitHub committed:
Update styles.css

Правки по вопросам в телеге ч.1

Alex88GitHub committed:
Правки

Корректировки по вопросам телеги ч2

Alex88GitHub committed:
Update index.html

Alex88GitHub committed:
Update index.html

Alex88GitHub committed:
Добавление класса для закрытия модального окна

Alex88GitHub committed:
Подготовка к защите

Удаление не используемого изображения (корзина). Корректировка оформления кода стилей.
